content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,123 @@
+# data status
+
+Show changes in data tracked by DVC between the last commit, the dvcfiles and
+the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+Shows changes that have differences between the last commit (`HEAD`) and
+committed data specified in [metafiles](/doc/user-guide/project-structure)
+files, uncommitted changes that have differences between that specified in the
+metafiles and the <abbr>workspace</abbr>, and the changes that are not in the
+cache but is specified in those metafiles.
+
+The first change shows what is already committed to dvc via `dvc add` or
```

Comment:
"fist chang", "second change" ... change for me is a specific item (e.g. data.xml changed), not a section or a kind/type of changes. I think we need to change language here. If we include the table of different section this whole paragraph is not needed?

Reply:
Need to rethink here. I don't find tables any better.

Reply:
I have no attachment to this, but in case it's helpful, here's the best I could do to quickly summarize:

The status is determined by comparing four different states:

1. Workspace data
2. Workspace dvcfiles
3. Git HEAD dvcfiles
4. Cache

* Untracked files: workspace data exists but is not tracked by DVC (no workspace dvcfiles) or Git.
* DVC uncommitted changes: the workspace data has changed from the workspace dvcfiles.
* DVC committed changes: the workspace dvcfiles have changed from the Git HEAD dvcfiles.
* DVC unchanged files: everything is up to date and matches between all states (actually it can show as both "unchanged" and "not in cache" right now 🤔 ).
* Not in cache: the workspace dvcfiles don't match any entry in the cache.

Reply:
A less technical take on DVC committed vs uncommitted:

* DVC uncommitted changes: workspace has changes that have not been DVC-committed.
* DVC committed changes: workspace has changes that have been DVC-committed but differ from the Git HEAD.

Are the other states unclear?

Are we over-complicating with indexes, dvcfiles, tables, etc.?

Reply:
> Are the other states unclear?
> 
> Are we over-complicating with indexes, dvcfiles, tables, etc.?

@shcheklein @jorgeorpinel Your feedback here would be great 🙏 , thank you!

Reply:
I've done another review. To me it's not about complexity or adding any tables/indexes on top of what we have. At the end it should be reasonable (simple, easy to understand, quick value - think about what users will be looking for first there). Right now it's repetitive, it's either goes to abstract or way too simple + it overlaps. We need to rethink (I put a some suggestions) how do we approach the Description section in the first place there.